website/azurerm.erb

```diff
@@ -42,6 +42,17 @@
               </ul>
             </li>
 
+            <li<%= sidebar_current("docs-azurerm-resource-app-service") %>>
+              <a href="#">App Service Resources</a>
```

Comment:
🤔 do you think it's worth making this "App Service (Web Apps) Resources", like we've done for CosmosDB/DocumentDB?